demos/vc_issuer/tests/issue_credential.rs

```diff
@@ -617,6 +617,11 @@ fn should_issue_credential_e2e() -> Result<(), CallError> {
                 prepared_context: prepared_credential.prepared_context,
             },
         )?;
+
+        println!(
```

Comment:
Did you intend to leave this here?

Reply:
Oops, not intended. Thanks for catching, removed now.